src/seedsigner/models/encode_qr.py

```diff
@@ -344,11 +344,13 @@ def derivation_to_keypath(path: str) -> list:
             return Keypath(arr, self.root.my_fingerprint, len(arr))
             
         origin = derivation_to_keypath(self.derivation)
+        self.use_info = None if self.network == "M" else CoinInfo(type=None, network=1)
```

Comment:
Should always use the constants instead of hard-coded strings.

```python
self.use_info = None if self.network == SettingsConstants.MAINNET else...
```